zokrates_proof_systems/src/solidity.rs

```diff
@@ -456,7 +456,7 @@ library Pairing {
         input[3] = p2.Y;
         bool success;
         assembly {
-            success := staticcall(sub(gas(), 2000), 6, input, 0xc0, r, 0x60)
+            success := staticcall(sub(gas(), 2000), 6, input, 0x80, r, 0x40)
```

Comment:
https://github.com/ethereum/go-ethereum/blob/master/core/vm/contracts.go#L446

`Bn256Add` takes 128 bytes (0x80) and outputs G1 which is 64 bytes (0x40)